bundle/cert-manager-operator/0.15.0-alpha.0/cert-manager-operator.v0.15.0-alpha.0.clusterserviceversion.yaml

```diff
@@ -0,0 +1,90 @@
+    mediatype: image/png
+  install:
+    spec:
+      clusterPermissions:
+      - serviceAccountName: cert-manager-operator
+        rules:
+        - apiGroups:
+          - '*'
+          resources:
+          - '*'
+          verbs:
+          - '*'
+      deployments:
+      - name: cert-manager-operator
+        spec:
+          replicas: 1
+          selector:
+            matchLabels:
+              app: cert-manager-operator
+          template:
+            metadata:
+              labels:
+                app: cert-manager-operator
+            spec:
+              serviceAccountName: cert-manager-operator
+              containers:
+              - name: cert-manager-operator
+                # Replace this with the built image name
+                image: registry.connect.redhat.com/jetstack/cert-manager-operator:v0.15.0-alpha.0
+                env:
+                - name: POD_NAME
+                  valueFrom:
+                    fieldRef:
+                      fieldPath: metadata.name
+                - name: OPERATOR_NAME
+                  value: "cert-manager-operator"
+    strategy: deployment
+  installModes:
+  - supported: false
+    type: OwnNamespace
+  - supported: false
+    type: SingleNamespace
+  - supported: false
+    type: MultiNamespace
+  - supported: true
+    type: AllNamespaces
+  keywords:
+  - TLS
+  - secrets
+  - security
+  - certificates
+  links:
+  - name: repository
+    url: https://github.com/jetstack/cert-manager
+  - name: conatinerImage
+    url: https://quay.io/jetstack/cert-manager:latest
+  maintainers:
+  - email: cert-manager-maintainers@jetstack.io
+    name: cert-manager team
+  maturity: alpha
```

Comment:
What does maturity here refer to? When does this change?

Reply:
This should refer to the application release, which for now is Alpha